Makefile

```diff
@@ -1,8 +1,14 @@
-.PHONY: help test clean dockerbuild dockerserve dockershell dockerprod geoip
+.PHONY: help test clean dockerbuild dockerserve dockershell dockerprod geoip ipproxy
+
 
-GEOIP_UPDATE = geoipupdate
 GEOIP_DIR = geoip
-GEOIP_CONF_FILE = $(GEOIP_DIR)/GeoIP.conf
+GEOIP_CITY_FILE = dbip-city-lite-2020-07.mmdb.gz
+GEOIP_COUNTRY_FILE = dbip-country-lite-2020-07.mmdb.gz
+GEOIP_CITY_DB_URL = https://download.db-ip.com/free/dbip-city-lite-2020-07.mmdb.gz
+GEOIP_COUNTRY_DB_URL = https://download.db-ip.com/free/dbip-country-lite-2020-07.mmdb.gz
```

Comment:
Looks like the `_FILE` variable should be used to build the URL

Reply:
The URLs will probably change while the filename probably won't since it needs to get picked up by the server. I'll make this more clear.